### PR TITLE
felt encoding: no leading zeros

### PR DESCRIPTION
**Pull request by ArielElp**
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/72)

### CODE REVIEW

File: api/starknet_api_openrpc.json [hunk 1]
```diff
@@ -967,9 +967,8 @@
             "FELT": {
                 "type": "string",
                 "title": "Field element",
-                "$comment": "A field element, represented as a string of hex digits",
-                "description": "A field element. Represented as 4 leading zero bits followed by 63 hex digits",
-                "pattern": "^0x0[a-fA-F0-9]{63}$"
+                "description": "A field element. represented by at most 63 hex digits",
+                "pattern": "$^0x(0|[a-fA-F1-9]{1}[a-fA-F0-9]{0,62})$"
```
Comment (THenry14) [inline api/starknet_api_openrpc.json:971]:
```suggestion
                "pattern": "^0x(0|[a-fA-F1-9]{1}[a-fA-F0-9]{0,62})$"
```

I think the `$` is here by mistake

**Review by ArielElp — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/72)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @THenry14)

___
*[`api/starknet_api_openrpc.json` line 971 at r1](https://reviewable.io/reviews/starkware-libs/starknet-specs/72#-NOd1yuNHRFXsYZ784-O-r1-971:-NOsORMv3aF7AxiVtozY:b-7ql1td) ([raw file](https://github.com/starkware-libs/starknet-specs/blob/2afc899043abac487bab03d23b227daf229573d2/api/starknet_api_openrpc.json#L971)):*
<details><summary><i>Previously, THenry14 (Wojciech Szymczyk) wrote…</i></summary><blockquote>

```suggestion
                "pattern": "^0x(0|[a-fA-F1-9]{1}[a-fA-F0-9]{0,62})$"
```

I think the `$` is here by mistake
</blockquote></details>

fixed

**Review by ArielElp — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> Dismissed @THenry14 from a discussion.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/72)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ArielElp)

**Review by anatgstarkware — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-specs/72#-:-NOtJHsK5LWay8N1aeAJ:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ArielElp)

**Pull request merged**